### ISSUE TITLE
How do I include all vendor libs into one and have it executed in browser

### ISSUE THREAD

**Issue by smac89** [webpack/webpack#9080]
**Do you want to request a *feature* or report a *bug*?**
Not sure if this is a bug or feature.

**What is the current behavior?**

**If the current behavior is a bug, please provide the steps to reproduce.**

What I am trying to do seems like a very easy task to me, but webpack is making it difficult.

I have an index.html which includes a bunch of javascript libraries at the top (jquery, bootstrap, respond, etc). 

What I did was create an entry with all those files; so something like this in my webpack.config.js:

```js
entry: {
        vendor: [
            'js/bootstrap.js',
            'js/jquery.1.8.3.min.js',
            'js/jquery.easing.1.3.js',
            'js/respond-1.1.0.min.js',
        ]
}
```

This produced a `vendor.js` file in my output. Now I thought I could just include that at the top of my html and it would work the same as when all those files where being included, but this does not seem to be the case.

Upon loading the page, I get errors complaining about **`jquery` not found** or some of the animations on the page are not working.

What do I need to do in order to simply have all those modules bundled into one file which can be included in a html document?

**What is the expected behavior?**
All js files bundled in vendor should be executed in the browser.

**If this is a feature request, what is motivation or use case for changing the behavior?**

**Please mention other relevant information such as the browser version, Node.js version, webpack version, and Operating System.**

```
Webpack version: 4.30.0
webpack-dev-server: 3.3.1
Node: version 11
Os: Arcolinux 64bit with latest kernel 5
```

**Comment by evilebottnawi** [comment]
@smac89 please use issue template, thanks

**Comment by smac89** [comment]
Problem solved.

First I removed the vendor entry from my webpack.config.js; then I copied all the vendor libraries into a main.js file and imported them all there. Then I added `main.js` as an entry point in webpack.config.js

**main.js**
```js
import 'vendor/jquery.easing.1.3';
import 'vendor/bootstrap';
import 'imports-loader?this=>window!vendor/respond-1.1.0.min';
```

Next I also moved all the scripts from the script tags in my html file into `main.js`. These scripts are the ones that needed jquery

Finally in the `plugins` section of my config, I added this plugin:

```js
new webpack.ProvidePlugin({
    $      : 'vendor/jquery.1.8.3.min',
    jQuery : 'vendor/jquery.1.8.3.min',
}),
```

After doing all that and executing the config, this solved the problem
